content-src/components/DiscoveryStreamComponents/DSEmptyState/_DSEmptyState.scss

```diff
@@ -0,0 +1,26 @@
+.section-empty-state {
+  border: $border-secondary;
+  border-radius: 4px;
+  display: flex;
+  height: 96px;
```

Comment:
Why do you need the fixed height here? Is that to line it up with other sections/elements? 

If so I would try to use a variable (like is that `$grid-unit` maybe?) or at least a comment

Reply:
yeah specs call for setting our empty state section height to 96px, will change to use $grid-unit

https://www.figma.com/file/0xZkmT86rSXbGRB4eIvrn0ts/New-Tab-Spec-for-Fx-v68?node-id=91%3A12